utilities/bitwarden.py

```
@@ -23,19 +27,43 @@ def _run_bws_command(args: list[str]) -> Any:
     access_token = os.getenv("ACCESS_TOKEN")
 
     if not access_token:
         raise MissingEnvironmentVariableError("Bitwarden client needs ACCESS_TOKEN environment variable set up")
 
-    _, stdout, _ = run_command(
-        command=["bws", "--access-token", access_token] + args,
+    return _run_bws_cli(access_token=access_token, args=args)
+
+
+@retry(wait_timeout=60, sleep=10)
+def _run_bws_cli(access_token: str, args: list[str]) -> Any:
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_run_bws_cli`

(ANN401)